### DISCUSSION TITLE
Github Student developer pack academic proof problem

### DISCUSSION THREAD

**Discussion by Ravindudeshitha** [community/community#86689] [GitHub Education]
### Select Topic Area

Question

### Body

"Upload proof of your academic status", for this proof if I click Process my application button it show "Processing information"  but it not submit

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Ravindudeshitha ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!
